roles/kubernetes-apps/meta/main.yaml

```diff
@@ -1,3 +1,20 @@
 dependencies:
+  - role: download
+    file: "{{ downloads.calico_policy }}"
+    tags: download
+    when: ( enable_network_policy is defined and enable_network_policy == True ) or
+      ( kube_network_plugin == 'canal' )
+  - role: download
+    file: "{{ downloads.netcheck_server }}"
+    tags: download
+    when: deploy_netchecker
+  - role: download
+    file: "{{ downloads.netcheck_agent }}"
+    tags: download
+    when: deploy_netchecker
+  - role: download
+    file: "{{ downloads.netcheck_kubectl }}"
+    tags: download
+    when: deploy_netchecker
```

Comment:
could you remove those from the roles/kubernetes/node/meta/main.yml then? please adjust tags as well, like specified above